File: C/URI/1046.c
#include <stdio.h>
 
int main() {
 
  int n1, n2, total;
        scanf("%d %d", &n1, &n2);
        total = 0;
        if (n1 == n2)
        {
                total = 24;
        }
        else if(n1 > n2)
        {
                while(n1 < 24)
                {
                        total = total + 1;
                        n1 = n1 + 1; 
                }
        total = total + n2;
        }
        else
        { 
                while(n1 < n2)
                {

                        total = total + 1;
                        n2 = n2 - 1;
                }
        }
        printf("O JOGO DUROU %d HORA(S)\n", total);

 
    return 0;
}

File: php/URI/1046.php
<?php

$linha = explode(' ', readline());


$a = $linha[0];
$b = $linha[1];


if($a == $b)
    {
       echo "O JOGO DUROU 24 HORA(S)\n";
    }
else if ($a > $b)
    {
        $total = $a - 24;
        $total = abs($total) + $b;
        echo "O JOGO DUROU ".$total." HORA(S)\n";
    }
else
    {
        
        
        $total = $b - $a;
        echo "O JOGO DUROU ".$total." HORA(S)\n";
    }


?>